### PR TITLE
calculation of m/z error in ppm

### PR DESCRIPTION
**Pull request by andreavicini**

### CODE REVIEW

File: R/matchMz.R [hunk 1]
```diff
@@ -228,7 +228,9 @@ MzRtParam <- function(tolerance = 0, ppm = 0, toleranceRt = 0) {
 #'
 #' @param ... currently ignored.
 #'
-#' @return [Matched] object representing the result
+#' @return [Matched] object representing the result. To evaluate each match the
+#' object contains the `ppm` variable with their m/z error in ppm (in addition
```
Comment (jorainer) [inline R/matchMz.R:232]: would maybe change that to:

contains the m/z error in ppm (variable `"ppm_error"`) as well as the absolute difference between the target and query m/z (variable `"score"`).

File: R/matchMz.R [hunk 2]
```diff
@@ -582,58 +584,67 @@ setMethod("matchMz",
...
+  absdiffs <- abs(diffs)
+  cls <- which(absdiffs <= (tolerance + ppm(queryMz, ppm)))
   if ("adduct" %in% colnames(target)){
     if (length(cls))
       data.frame(query_idx = queryIndex,
                  target_idx = target$index[cls],
                  adduct = target$adduct[cls],
-                 score = diffs[cls])
+                 score = absdiffs[cls],
+                 ppm = diffs[cls] / target[cls, "mz"] * 10^6)
```
Comment (jorainer) [inline R/matchMz.R:596]: maybe better to call that column `ppm_error`? Then it would be clear what exactly it contains.

**Review by jorainer — CHANGES_REQUESTED** [review]
Thanks Andrea! Looks good to me. I would maybe just rename the variable.

File: R/matchMz.R (hunk 1 above)
Comment (andreavicini) [reply R/matchMz.R]: much better. Thanks!

File: R/matchMz.R (hunk 2 above)
Comment (andreavicini) [reply R/matchMz.R]: Yes, I agree. I thought that too but I saw `ppm` on your message and that maybe it was to be concise. I changed it to `ppm_error'

**Review by jorainer — CHANGES_REQUESTED** [review]
Perfect! Thanks Andrea! One very last request, please:
- increment the version of the package (the "z" part in "x.y.z"
- add this version also to the NEWS.md and descbribe these changes in the NEWS.

**Comment by andreavicini** [comment]
I should have done it (also added my ORCID since now I have it, hope it's ok)!

**Review by jorainer — APPROVED** [review]
Perfect! Thanks! I'll merge and push to BioC.

**Pull request merged**